EU4toV2/Source/EU4toV2Converter.cpp

```diff
@@ -43,9 +44,15 @@ void ConvertEU4ToVic2(const string& EU4SaveFileName)
 	mappers::IdeaEffectMapper ideaEffectMapper(ideaEffectsFile);
 	ideaEffectsFile.close();
 
+	std::ifstream techGroupsFile("techGroups.txt");
+	mappers::TechGroupsMapper techGroupsMapper(techGroupsFile);
+	LOG(LogLevel::Info) << "Closing techgroups.";
```

Comment:
Was this meant to be permanent logging? If not, remove.

Reply:
removed.